### PR TITLE
Enable python tests.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +40,5 @@ script:
     - ./lint.sh
     - ../node_modules/.bin/mocha static/build_test.js
     - cd ..
-    - gofmt -l .
```
Comment (spxtr) [inline .travis.yml]: This doesn't actually do anything.

File: jenkins/bootstrap_test.py [hunk 2]
```diff
@@ -130,6 +130,32 @@ def testZombie(self):
...
+        self.assertListEqual(
+            bootstrap.pull_numbers('master:abcd,123:qwer,124:zxcv'),
+            ['123', '124'])
+
+    def testPullRef(self):
+        def test_tuplist_equal(a, b):
+            self.assertEqual(len(a), 2)
+            self.assertListEqual(a[0], b[0])
+            self.assertListEqual(a[1], b[1])
+        test_tuplist_equal(bootstrap.pull_ref('123'),
```
Comment (rmmh) [inline jenkins/bootstrap_test.py]: you should be able to just do self.assertEqual

File: .travis.yml [hunk 3]
```diff
@@ -25,18 +24,13 @@ install:
 # Based on https://github.com/travis-ci/travis-ci/issues/738#issuecomment-11179888
     - wget -nv https://storage.googleapis.com/appengine-sdks/featured/${GAE_ZIP}
     - unzip -q ${GAE_ZIP} -d ${HOME}
-    - pip install -r gubernator/test_requirements.txt
```
Comment (rmmh) [inline .travis.yml]: we're still running gubernator tests with nose. don't get rid of this line.

File: jenkins/bootstrap_test.py (hunk 2 above)
Comment (spxtr) [inline jenkins/bootstrap_test.py]: Done.

File: .travis.yml (hunk 3 above)
Comment (spxtr) [inline .travis.yml]: Done.

**Comment by rmmh** [comment]
LGTM

**Comment by spxtr** [comment]
I'll get rid of more in the next PR.

**Pull request merged**